UI/src/authorizationpolicies/linearAssetAuthorizationPolicy.js

```diff
@@ -5,7 +5,10 @@
     var me = this;
 
     this.formEditModeAccess = function(selectedAsset) {
-      return (me.isMunicipalityMaintainer() && !me.isState(selectedAsset) && me.hasRightsInMunicipality(selectedAsset.municipalityCode)) || (me.isElyMaintainer() && me.hasRightsInMunicipality(selectedAsset.municipalityCode)) || me.isOperator();
+      var isMunicipalityMaintainerAndHaveRights = me.isMunicipalityMaintainer() && !me.isState(selectedAsset) && me.hasRightsInMunicipality(selectedAsset.municipalityCode);
+      var isElyMaintainerAndHasRights = me.isElyMaintainer() && me.hasRightsInMunicipality(selectedAsset.municipalityCode);
+
+      return  me.isMunicipalityExcluded(selectedAsset) ||  ( isMunicipalityMaintainerAndHaveRights || isElyMaintainerAndHasRights || me.isOperator() );
```

Comment:
It's really needed this modification?

It's because this method it's overrided at linearStateRoadAuthorizationPolicy.js file so I think the only modification needed it's the applied bellow.

Because with that they will apply these rules in all linear assets and we probably don't want that.

Reply:
lengthLimit and widthLimit uses LinearAssetAuthorizationPolicyWith Suggestion and this object calls linearAsset AuthorizationPolicy to validate if a form should or not be able to edit